base/bucket_test.go

```diff
@@ -276,3 +275,19 @@ func TestGetPoolName(t *testing.T) {
 	fakeBucketSpec.PoolName = "Liverpool"
 	assert.Equal(t, "Liverpool", fakeBucketSpec.GetPoolName())
 }
+
+func TestGetViewQueryTimeoutMs(t *testing.T) {
```

Comment:
If we test `GetViewQueryTimeout` instead, we can get coverage for both functions.

Reply:
This has been fixed; TestGetViewQueryTimeout included and TestGetViewQueryTimeoutMs is removed.